gitgud/skills/level_builder.py

```diff
@@ -43,6 +43,7 @@ def _setup(self):
 
     def setup(self):
         self._setup()
+        self.mark_visited()
```

Comment:
We do not want to mark this as visited if it has already been completed. Can you add checks in this method to make sure we never overwrite a "complete" or "partial"? Likewise, "partial" should never be overwritten by anything but "complete"